### ISSUE TITLE
How to configure In-App Messaging to show the message after splash screen

### ISSUE THREAD

**Issue by oth-libs** [firebase/firebase-android-sdk#1410]
### [REQUIRED] Step 2: Describe your environment

  * Android Studio version: 3.6.1
  * Firebase Component: In-App Messaging
  * Component version: firebase-inappmessaging-display-ktx:19.0.4

### [REQUIRED] Step 3: Describe the problem
I can successfully receive a message on my splash activity, but in reality I need to show in a later activity, so in firebase in the Scheduling section I removed all the events and added a new one "login_activity_opened", which I trigger when the login activity is opened.
But the event is always shows when I open the app, in the splash activity screen.

#### Steps to reproduce:
as shown in the description

#### Relevant Code:
How I trigger the event:
```
FirebaseAnalytics.getInstance(context).logEvent("login_activity_opened", trackEvent.getEventValues()?.toBundle())
FirebaseInAppMessaging.getInstance().triggerEvent("login_activity_opened")
```

**Comment by JasonAHeron** [comment]
Hey @oth-libs is your campaign set to trigger on this event in the firebase console? by default campaigns trigger on app foreground which seems to be what you're experiencing.

**Comment by Viraj7009** [comment]
Hi @JasonAHeron  I think what @oth-libs is facing is similar to this issue https://github.com/firebase/firebase-android-sdk/issues/1092. I commented there to see if there's any solution for this as I didn't find any yet. Any help would be appreciated. Thanks!

**Comment by JasonAHeron** [comment]
This will be fixed in our next release 19.0.7
